app/adapters/alma_adapter.rb

```diff
@@ -48,6 +48,42 @@ def get_items_for_bib(id)
     bib_item_set.holding_summary
   end
 
+  # @param id [String]. e.g id = "991227850000541"
+  # @return [String] date record was created e.g. "2019-10-18Z"
+  def get_catalog_date(id)
```

Comment:
Do we need the `get_catalog_date` method or can we just remove this?

Reply:
We don't need _need_ the `get_catalog_date(id)` method but it keeps the test simple (i.e. allow us to use similar fixtures to existing tests that also emulate Alma API responses). So I would be in favor for keeping it.

Reply:
Good point. Done.